tests/Silverback.Integration.Tests.E2E/MQTT/SerializationTests.cs

```diff
@@ -148,7 +148,7 @@ public async Task JsonSerializer_WithHardcodedMessageType_MessageTypeHeaderIgnor
                 {
                     Configuration =
                         ((MqttClientConfigBuilder)new MqttClientConfigBuilder()
-                            .WithClientId("e2e-test")
+                            .WithClientId("e2e-test-helper")
```

Comment:
Why? 😮

Reply:
Otherwise I get an error because we have the same client ID, but different EventHandlers for the endpoints.
I think this is the desired behavior. Here we have a rather unusual use case (constructing and endpoint manually instead of getting it from the application). Therefore I changed the client id to avoid the error.